__tests__/app.test.js

```diff
@@ -119,6 +119,29 @@ describe('GET /api/articles/:article_id/comments', ()=> {
         })       
     })
 })
+describe('POST /api/articles/:article_id/comments', () => {
+    test('server responds with 201 status and the posted comment', () => {
+        const newComment = { username: 'lurker',
+        body: 'opinions.tm'}
+        return request(app)
+        .post('/api/articles/1/comments')
+        .expect(201)
+        .send(newComment)
+        .then(response => {
+            expect(response.body).toMatchObject({
+                posted_comment: {
+                    comment_id: expect.any(Number),
+                    body: 'opinions.tm',
+                    article_id: 1,
+                    author: 'lurker',
+                    votes: expect.any(Number),
+                    created_at: expect.any(String)
+                }
+            })
+                       
+        })
+    })
+})
```

Comment:
It is possible for our client to send unnecessary properties on the post object. In this case we need another test. 201 - ignores unnecessary properties